cmd/notification.go

```diff
@@ -634,6 +634,47 @@ func (sys *NotificationSys) ReloadPoolMeta(ctx context.Context) {
 	}
 }
 
+func (sys *NotificationSys) StopRebalance(ctx context.Context) {
```

Comment:
I am not sure why stop rebalance needs to be sent to all nodes

Reply:
It doesn't have to be sent to all nodes. We need to only notify the first nodes on each pool participating in rebalance. In its current form, non-participating nodes have nothing to do, so it is the same in effect.